src/checklist/ChecklistMultiSignOrVerify.tsx

```diff
@@ -6,14 +6,10 @@ import { ProcosysApiService } from '../services/procosysApi';
 import { COLORS } from '../style/GlobalStyles';
 
 const Wrapper = styled.div`
```

Comment:
This does need to be position: fixed and fullscreen in the MC app. Lets discuss some solutions for this.

<img width="645" alt="Screenshot 2021-10-07 at 10 32 13" src="https://user-images.githubusercontent.com/8993868/136351001-2350590b-1df5-4d72-b7b3-701ba9599485.png">
.